### ISSUE TITLE
0.14.4 failed some parts of the release

### ISSUE THREAD

**Issue by giarc3** [IronCoreLabs/ironoxide-swig-bindings#197]
Successful release kickoff: https://github.com/IronCoreLabs/ironoxide-swig-bindings/actions/runs/2740960283
Failed GitHub release: https://github.com/IronCoreLabs/ironoxide-swig-bindings/actions/runs/2740974253
Failed Java release: https://github.com/IronCoreLabs/ironoxide-swig-bindings/actions/runs/2740974256
Successful Android release: https://github.com/IronCoreLabs/ironoxide-swig-bindings/actions/runs/2740974261

The common error message is `gpg: no default secret key: No secret key`, but the step to import the key succeeded, so it's possible that the key is expired or that something about gpg commands changed in the ~2 years since this was last run.

**Comment by cjyar** [comment]
```
~ # gpg --list-keys
/root/.gnupg/pubring.kbx
------------------------
pub   ed25519 2020-03-03 [C] [expired: 2021-03-03]
      83F949C6E7E2F0A295642DEE62F57B1B87928CAC
uid           [ expired] IronCore Labs (Package Signing) <ops@ironcorelabs.com>
```

**Comment by giarc3** [comment]
Several parts of the build process were fixed for v0.14.7 release
